test/helpers/message.spec.ts

```diff
@@ -161,8 +184,35 @@ describe('buildFillOrderMsg', () => {
 
     expect(actual).toMatch(
       new RegExp(
-        `Fill the order here: ${baseUrl}/trade/${buyTokenParam}-${sellTokenParam}\\?sell=0\\.000000000000009802\\&buy=0\\.00000000000000001`,
+        `Fill the order here: ${baseUrl}/trade/${buyTokenParam}-${sellTokenParam}\\?sell=0\\.000000000000009802\\&buy=0\\.00000001`,
       ),
     )
   })
 })
+
+describe('newOrderMessage', () => {
+  const baseUrl = 'http://dex.gnosis.io/'
+
+  test('unlimited order', () => {
+    // GIVEN: An order staring 2min 10 seg ago. With expiring date on next date 12am GMT
+    Date.now = jest.fn().mockReturnValue(new Date('2020-02-24T00:02:10.000'))
```

Comment:
You are using a fixed date here. Will it always be tomorrow?

Reply:
Yes, that's the point of mocking it.

I added more of this in this other PR:
https://github.com/gnosis/dex-telegram/pull/159

It says: Mock time in tests to be able to test using equals for example in relative times. i.e. expect(active).toEqual('Today at 3:10 AM GMT, in 3 hours') this way the test is more readable and is still repeatable.